docs/platforms/javascript/common/feature-flags/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+description: With Feature Flags, Sentry tracks flag evaluations in your application and reports their state on error. Sentry will also record an audit log of feature flag changes and report any suspicious changes that may have triggered an error.
+---
+
+<PlatformContent includePath="feature-flags/prerelease-alert" />
+
+## Prerequisites
+
+* You have the <PlatformLink to="/">JavaScript SDK installed</PlatformLink> (version 8.41.0-beta.1).
+
+## Enable Evaluation Tracking
+
+Evaluation tracking requires enabling an SDK integration. Integrations are provider specific. Documentation for supported providers is listed below.
+- [OpenFeature](/platforms/javascript/configuration/integrations/openfeature/)
+- [LaunchDarkly](/platforms/javascript/configuration/integrations/launchdarkly/)
```

Comment:
these links arn't working either

Reply:
yeah that page is broken i think.  the links should work when its fixed.